Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contests-filters/ContestFilters.tsx

```diff
@@ -54,32 +55,56 @@ const ContestFilters = ({ onFilterClick }: IContestFiltersProps) => {
         [ possibleFilters, onFilterClick ],
     );
 
-    const getRenderFilterItemFunc = useCallback(
-        (type: FilterType) => ({ id, name }: IFilter) => {
-            // TODO: investigate why filters change ids
-            //  and use id instead of name for checking if filter is selected
+    const renderStatusFilterItem = useCallback(
+        (buttonType: ButtonType, btnClassName: string,name: string, id: number) => (
+            <Button
+                    type={buttonType}
+                    onClick={() => handleFilterClick(id)}
+                    className={btnClassName + styles.btnSelectFilter}
+                    text={name}
+                    size={ButtonSize.small}
+                />
+        ),
+        [ handleFilterClick ],
+    );
+    
+    const renderStrategyFilterItem = useCallback(
+        (buttonType: ButtonType, btnClassName: string, name: string, id: number)=> (
+            <div className={styles.strategyHeader}>
+                <div className={styles.tooltip}>
+                    <span className={styles.tooltipElement}>{name}</span>
+                </div>
+                <Button
+                        type={buttonType}
+                        onClick={() => handleFilterClick(id)}
+                        className={styles.strategyElementClassName}
+                        text={name}
+                        size={ButtonSize.small}
+                    />
+            </div>
+        ),
+        [ handleFilterClick ],
+    );
+    
+    const getRenderFilterItem = useCallback(
```

Comment:
Why is this renamed? `getRenderFilterItemFunc` clearly says this method returns a func